util/mk-fipsmodule-cnf.pl

```diff
@@ -50,9 +50,9 @@
 drbg-no-trunc-md = $drgb_no_trunc_dgst
 module-mac = $module_mac
 hkdf-digest-check = $kdf_digest_check
-tls13-kdf-digest-check = $kd_digest_check
+tls13-kdf-digest-check = $kdf_digest_check
```

Comment:
@paulidale @slontis Any reason we don't have `use strict` in this file?

Reply:
No idea.  @levitte?